wrangling.Rmd

```diff
@@ -1153,46 +1130,44 @@ Now that we've shown you the pipe operator as an alternative to storing
 temporary objects and composing code, does this mean you should *never* store 
 temporary objects or compose code? Not necessarily! 
 There are times when you will still want to do these things. 
-For example, you might store temporary objects 
-because you want to save prepared data before feeding it into a plot function 
+For example, you might store a temporary object before feeding it into a plot function 
 so you can iteratively change the plot without having to
-redo all of your data transformations each time you create a new plot.
-Additionally, piping many functions can be overwhelming and difficult to debug, 
-so you may also want to store a temporary object midway through 
-and pipe that into more functions after that.
+redo all of your data transformations. 
+Additionally, piping many functions can be overwhelming and difficult to debug;
+you may want to store a temporary object midway through to inspect your result
+before moving on with further steps.
 
 ## Aggregating data with `summarize`
 
-### Calculating summary statistics on columns
+### Calculating summary statistics on whole columns
 
 As a part of many data analyses, we need to calculate a summary value for the
-data (a *summary statistic*). 
+data (a *summary statistic*). \index{summarize}
 Examples of summary statistics we might want to calculate 
 are the number of observations, the average/mean value for a column, 
 the minimum value, etc. 
 Often times, 
 this summary statistic is calculated from the values in a data frame column, 
-or columns, as shown in Figure \@ref(fig:summarize)).
+or columns, as shown in Figure \@ref(fig:summarize).
 
-```{r summarize, echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "`summarize` is useful for calculating summary statistics on one, or more, column(s). In it's simplest use case, it creates a new data frame with a single row containing the summary statistic(s) for each column being summarized. The darker, top row of each table represents the column headers.", out.width="100%", fig.retina = 2}
+```{r summarize, echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "`summarize` is useful for calculating summary statistics on one or more column(s). In its simplest use case, it creates a new data frame with a single row containing the summary statistic(s) for each column being summarized. The darker, top row of each table represents the column headers.", out.width="100%", fig.retina = 2}
```

Comment:
In the figure caption `summarize` displays as "summarize" -- Also I think it would help to add numbers to this table. And then you can use the example given in the paragraph above as part of this picture. e.g. with just 5 number or something find the mean or max or something.

Reply:
Totally agree that it would be good to add numbers to these figs. But this is out of scope for my edit pass, and I don't want to block the PR on it. I'll create an isue to think about it for version 2 of the book.